lib/plugins/plugin/install/install.js

```diff
@@ -109,6 +109,14 @@ class PluginInstall {
 
   addPluginToServerlessFile() {
     return this.getServerlessFilePath().then(serverlessFilePath => {
+      if (_.last(_.split(serverlessFilePath, '.')) === 'js') {
```

Comment:
Need the same process in `plugin/uninstall/uninstall.js` as well. Otherwise, a plugin can't be removed from serverless.js when running `sls plugin uninstall`

Reply:
Good catch, added log message for uninstall as well 👍